File: Java/areal.java
/*
 *	Problem: "A Real Challenge"
 */

import java.util.Scanner;
import java.lang.Math;

public class areal {
	
	/*-----main-----*/
	public static void main(String[] args) {		
		Scanner sc = new Scanner(System.in);
		
		long area = sc.nextLong();
		double perimeter = (Math.sqrt(area) * 4);

		System.out.println(perimeter);
	} // end main
	
} // end class

File: Ruby/areal.rb
=begin
 *
 *	Problem:		A Real Challenge
 *	Problem ID:	areal
 *	url:				https://open.kattis.com/problems/areal
 *	Solution:		Ruby
 *
=end

class Solver
	def solve
		area = gets.chomp.to_i
		total_length = (Math.sqrt(area) * 4)
		puts total_length
	end
end

solver = Solver.new
solver.solve
